src/pages/profile/index.ts

```diff
@@ -165,35 +185,150 @@ const profileItems = [
+    inputIconClass: null,
+    urlImg: null,
+    attr: {
+        class: 'input'
+    },
+    events: {
+        input: (e: Event) => validateInput(e),
+        focus: (e: Event) => validateInput(e),
+        blur: (e: Event) => validateInput(e),
+    }
+});
+
+const newPasswordInput = new Input({
+    inputClass: 'input_profile',
+    type: 'text',
+    placeholder: 'НОВЫЙ ПАРОЛЬ',
+    inputName: 'new_password',
+    inputIconClass: null,
+    urlImg: null,
+    attr: {
+        class: 'input'
+    },
+    events: {
+        input: (e: Event) => validateInput(e),
+        focus: (e: Event) => validateInput(e),
+        blur: (e: Event) => validateInput(e),
+    }
+});
 
 const profilePage = new Profile({
     src: imgUrl,
-    title: 'Джеки Чан',
+    title: '',
     dataList: profileItems,
+    popup: new Popup({
+        closeIcon: closeIcon,
+        inputs: [
+            new Input({
+                inputClass: 'input__item_popup',
+                type: 'text',
+                placeholder: 'СТАРЫЙ ПАРОЛЬ',
+                inputName: 'oldPassword',
+                inputIconClass: null,
+                urlImg: null,
+                attr: {
+                    class: 'input input_popup'
+                },
+                events: {
+                    input: (e: Event) => validateInput(e),
+                    focus: (e: Event) => validateInput(e),
+                    blur: (e: Event) => validateInput(e),
+                }
+            }),
+            new Input({
+                inputClass: 'input__item_popup',
+                type: 'text',
+                placeholder: 'НОВЫЙ ПАРОЛЬ',
+                inputName: 'newPassword',
+                inputIconClass: null,
+                urlImg: null,
+                attr: {
+                    class: 'input input_popup'
+                },
+                events: {
+                    input: (e: Event) => validateInput(e),
+                    focus: (e: Event) => validateInput(e),
+                    blur: (e: Event) => validateInput(e),
+                }
+            })
+        ],
+        button: new Button({
+            text: 'СОХРАНИТЬ',
+            attr: {
+                class: 'button'
+            },
+            events: {
+                click: (e:Event) => {
+                    const inputs: NodeList | null = document.querySelectorAll('.input__item_popup');
+                    let data = {
+                        oldPassword: '',
+                        newPassword: ''
+                    };
+                    inputs.forEach(input => {
+                        const inputName = (input as HTMLInputElement).name;
+                        if ((input as HTMLInputElement).value) {
+                            data[inputName] = (input as HTMLInputElement).value;
+                            (input as HTMLInputElement).value = '';
+                        }
+                    });
+                    UserController.changePassword(data);
```

Comment:
```suggestion
                                        const inputs = document.querySelectorAll<HTMLInputElement>('.input__item_popup');
                    let data = {
                        oldPassword: '',
                        newPassword: ''
                    };
                    inputs.forEach(input => {
                        const inputName = input.name;
                        if (input.value) {
                            data[inputName] = input.value;
                            input.value = '';
                        }
                    });
                    await UserController.changePassword(data);
```

Нужно стараться избегать явного приведения типов